docs/ref/migration-operations.txt

```diff
@@ -60,6 +60,11 @@ from the standard ``models.Model``.
 The first manager in the list will be the default manager for this model during
 migrations.
 
+.. versionadded:: 3.2
+
+``metaclass`` takes a class that is the metaclass of the model if the model's
+bases don't have related metaclasses. Used for constructing the model's class.
```

Comment:
The `metaclass` argument should be described in the `CreateModel()` description not in a `versionadded` annotation.

```suggestion
.. versionadded:: 3.2

    The ``metaclass`` argument was added.
```